### PR TITLE
Fix opening market when app is uninstalled

### PR DESCRIPTION
**Pull request by ButtersHub**
On Android - Currently when app is not installed a default appChooser is opened instead of opening google play or weblink.

### CODE REVIEW

File: android/src/main/java/cl/json/social/SingleShareIntent.java [hunk 1]
```diff
@@ -52,8 +52,9 @@ public void open(ReadableMap options) throws ActivityNotFoundException {
                 //  open web intent
                 this.setIntent(new Intent(new Intent("android.intent.action.VIEW", Uri.parse(url))));
             }
-        }
-        //  configure default
-        super.open(options);        
+        } else{
```
[the remark this answers is not in the document]
Comment (enieber) [reply android/src/main/java/cl/json/social/SingleShareIntent.java]: the indentation is not ok @jgcmarins ?

**Comment by jgcmarins** [comment]
hey @ButtersHub could you help us on this to get merged?

**Comment by ButtersHub** [comment]
@jgcmarins, what else missing in this PR?

**Comment by jgcmarins** [comment]
yep, right now \o/

**Comment by vinayjohn183** [comment]
Is the fix available now in the package?

**Pull request merged**